### PR TITLE
Iss328

### PR DESCRIPTION
**Pull request by awunderground**

### CODE REVIEW

File: geographic-crosswalks/data/county-populations.csv
Comment (jwalsh28) [inline geographic-crosswalks/data/county-populations.csv:1]: This version still has the low number of counties for 2021

**Review by jwalsh28 — CHANGES_REQUESTED** [review]

File: 10_construct-database/61_construct_place_all.qmd [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+  # 09 employment
+  here("09_employment/data/final/metrics_employment_place_all_longitudinal.csv")
+  # now age ratio for places
+)
+
+# variables erroneously included in the data sets
+extra_variables <- c(
+  "state_place" # race-ethnicity exposure
+)
+
```
[the remark this answers is not in the document]
Comment (awunderground) [reply 10_construct-database/61_construct_place_all.qmd:152]: I just added everything else. I'm waiting on housing.

File: geographic-crosswalks/data/county-populations.csv
Comment (awunderground) [reply geographic-crosswalks/data/county-populations.csv:1]: I think I fixed that now!

File: 10_construct-database/11_construct_county_all.qmd [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+  count(variable, quality) |>
+  filter(!is.na(quality)) |>
+  print(n = 100)
+
+db |>
+  select(year, ends_with("_quality")) |>
+  pivot_longer(-year, names_to = "variable", values_to = "quality") |>
+  count(year, quality) |>
+  print(n = 100)
+
```
[the remark this answers is not in the document]
Comment (jwalsh28) [reply 10_construct-database/11_construct_county_all.qmd:289]: I forgot this starts by joining onto the population file which has 2023 information, this is fine

**Review by jwalsh28 — APPROVED** [review]
These look ready to rock

**Pull request merged**